.vscode/settings.json

```diff
@@ -2,12 +2,16 @@
   "dart.lineLength": 120,
   "yaml.format.singleQuote": true,
   "dart.flutterSdkPath": ".fvm/flutter_sdk",
+  "files.insertFinalNewline": true,
   // Remove .fvm files from search
   "search.exclude": {
     "**/.fvm": true
   },
   // Remove from file watching
```

Comment:
I hate that they allow comments inside json

Reply:
They also allow trailing commas :eyes: